doc/source/user-guide/configuration.rst

```diff
@@ -0,0 +1,78 @@
+.. _ref_user_guide_configuration:
+
+Basic configuration
+===================
+
+Add the following to your `conf.py` file to use the Ansys Sphinx theme:
+
+.. code-block:: python
+
+   html_theme = "ansys_sphinx_theme"
+
+From ansys sphinx theme you can use the following features:
+
+1. PyAnsys and Ansys logos
```

Comment:
Let's avoid numbering these subsections. It is annoying to maintain if changing the order at some point.